actions/generate-changelog/action.yaml

```diff
@@ -0,0 +1,62 @@
+name: "Generate changelog"
+description: "Generate changelog after release"
+inputs:
+  # required inputs
+  changelog-file:
+    description: Path to the Changelog.md file
+    required: true
+  github-token:
+    description: "The GitHub token for committing the changes."
+    required: true
+  new-tag:
+    description: Version after bump
+    required: true
+  old-tag:
+    description: Previous version
+    required: true
+
+  # unrequired inputs
+  config:
+    description: Path to the changelog config JSON file
+    required: false
+    default: "./.github/changelog-config.json"
+  output:
+    description: "Name of the output file for the generate changelog step"
+    required: false
+    default: "changes.md"
+  fetchReviewers:
+    description: "Will enable fetching the users/reviewers who approved the PR"
+    required: false
+    default: "false"
+  fetchReleaseInformation:
+    description: "Will enable fetching additional release information from tags."
+    required: false
+    default: "false"
+  commitMode:
+    description: "Special configuration for projects which work without PRs."
+    required: false
+    default: "false"
+
+runs:
+  using: "composite"
+  steps:
+    - name: "Create changelog"
+      id: build_changelog
+      uses: mikepenz/release-changelog-builder-action@v3.7.0
+      with:
+        token: "${{ inputs.github-token }}"
+        configuration: "${{ inputs.config }}"
+        fromTag: "${{ inputs.old-tag }}"
+        toTag: "${{ inputs.new-tag }}"
+        outputFile: "${{ inputs.output }}"
+        fetchReviewers: "${{ inputs.fetchReviewers }}"
+        fetchReleaseInformation: "${{ inputs.fetchReleaseInformation }}"
+        commitMode: "${{ inputs.commitMode }}"
+
+    - name: Update github Changelog
```

Comment:
Do we need to check `outputs.failed`?

Reply:
what do you mean?

Reply:
the `updste-step`  wont be executed if the  `created-one`. so i dont think we need to explicitly check that